js/AudioWorkletGlobalScope.js

```diff
@@ -0,0 +1,138 @@
+const {
+  parentPort,
+  workerData,
+} = require('node:worker_threads');
+
+const {
+  run_audio_worklet,
+} = require('../load-native.cjs');
+
+const kHiddenOptions = Symbol('node-web-audio-api:hidden-options');
+const kWorkletInputs = Symbol.for('node-web-audio-api:worklet-inputs');
+const kWorkletOutputs = Symbol.for('node-web-audio-api:worklet-outputs');
+const nameProcessorCtorMap = new Map();
+const paramDescriptorRegisteredMap = new Map();
+let loopStarted = false;
+
+console.log(workerData);
+const { workletId } = workerData;
+console.log('coucou', workletId)
+
+function isIterable(obj) {
+  // checks for null and undefined
+  if (obj === null || obj === undefined) {
+    return false;
+  }
+  return typeof obj[Symbol.iterator] === 'function';
+}
+
+function runLoop() {
+  // console.log(workerData.workletId);
+  // block until we need to render a quantum
+  run_audio_worklet(workerData.workletId);
```

Comment:
Probably use the const, unpacked value here?
```suggestion
  run_audio_worklet(workletId);
```

Reply:
(sorry, I don't have the reflex of github suggestions yet, anyway this is done!)